app/post.go

```diff
@@ -616,11 +616,11 @@ func (a *App) GetPostsEtag(channelId string) string {
 }
 
 func (a *App) GetPostsSince(channelId string, time int64) (*model.PostList, *model.AppError) {
-	result := <-a.Srv.Store.Post().GetPostsSince(channelId, time, true)
```

Comment:
You can just `return a.Srv.Store.Post().GetPostsSince(channelId, time, true)` here.